### ISSUE TITLE
Multi-root config

### ISSUE THREAD

**Issue by scriptcoded** [vscode-icons/vscode-icons#1650]
- [x] I'm sure this issue is not a _duplicate_.

So, multi-root workspaces works as intended - neat! However, I believe there is something missing. There doesn't seem to be any way to configure icons per-root-folder. Say for example I have a workspace with the following root folders:
- Root
- Vue.js
- Backend

What if I'd like the Vue.js-folder to have the Vue.js logo? As far as I can tell, there is no way to do this as of right now. Howerever, IMO, there should absolutely be. Perhaps matching by folder name would be a thing?

**Comment by JimiC** [comment]
You are right. What you are trying to do is not achievable. This is a limitation of `vscode` itself. You can have only **one** default root folder icon. You can change it via the icon customization feature but then all root folders will have that icon.

**Comment by scriptcoded** [comment]
@JimiC Ouch, that's sad. Would this perhaps be something to raise over at [Microsoft/vscode](https://github.com/Microsoft/vscode/issues)?

**Comment by rcdsp** [comment]
What about separated but simultaneously opened workspaces? Having two projects open also mix icon presets. I dont know if its posible to at least detect the active window and try to quickly update icons if to the active window workspace settings for vsicons preset.

I ran this example over a local Windows and a WSL instance but its the same behavior with both instances being local or on WSL.

![vsicons-workspace-preset](https://user-images.githubusercontent.com/25913132/77116098-311b5400-69f5-11ea-8b0d-bb20b78898b1.gif)
